src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -75,13 +75,13 @@ public async Task Initialize(ContainerComponent containerComponent, Recoverabili
             }
         }
 
-        public void Start()
+        public async Task Start()
         {
             foreach (var receiver in receivers)
             {
                 try
                 {
-                    receiver.Start();
+                    await receiver.Start().ConfigureAwait(false);
```

Comment:
@timbussmann we could make this use `Task.WhenAll` just like the `Stop` but I guess that would be a behavior change?

Reply:
So the consequence would be that multiple instance of the transport infra would be starting at the same time. That should not be a problem assuming the code in the transport is thread-safe :trollface:

Reply:
Yea pretty sure it would be ok but we can change that when we fix the `IPushMessages.Start` to be task returning in the next major (I'll raise an issue for this shortly)